src/server.rs

```diff
@@ -131,7 +131,6 @@ where
     pub async fn run(self, addr: impl Into<SocketAddr>) {
         let (addr, fut) = self.bind_ephemeral(addr);
         let span = tracing::info_span!("Server::run", ?addr);
-        tracing::info!(parent: &span, "listening on http://{}", addr);
```

Comment:
can we maintain `listening on http://{}" instead of `Server::run` as it was the original log?

Reply:
To be honest, I have no idea what lines 133 and 135 do (not much experience with Rust or this codebase). Line 134 can be cleanly removed without any other code changes. Would changing L133 to `let span = tracing::info_span!("Listening on", ?addr);` while removing L134 work?